<launch>
    <arg name="drone_id" default="0"/>
    <!-- Start Ftxui server -->
    <node pkg="ftxui_ros" name="drone_$(arg drone_id)_ftxui_ros" type="ftxui_ros_node" output="screen">
        <remap from="mavros/state" to="/iris_$(arg drone_id)/mavros/state"/>
        <remap from="mavros/vision_odom/odom" to="/iris_$(arg drone_id)/mavros/vision_odom/odom"/>
        <remap from="px4ctrl_mode" to="/iris_$(arg drone_id)/px4ctrl_mode"/>
        <remap from="egoPlaner_mode" to="/iris_$(arg drone_id)/egoPlaner_mode"/>
    </node>
</launch>
    